transmogrify/dexterity/converters.py

```diff
@@ -382,6 +383,33 @@ def __call__(self, value, filestore, item,
         return value
 
 
+@implementer(IDeserializer)
+@adapter(IDatetime)
+class DatetimeDeserializer(object):
+
+    def __init__(self, field):
+        self.field = field
+
+    def __call__(self, value, filestore, item,
+                 disable_constraints=False, logger=None):
+        if isinstance(value, basestring):
+            value = datetime.strptime(value, '%Y-%m-%d %H:%M:%S')
```

Comment:
For example with collective.jsonify, we might have differently formatted dates, e.g. ``2004/12/30 00:20:00 GMT+1``.

I propose to let zope's `DateTime` handle everything and to not define a format:
```python
>>> from DateTime import DateTime
>>> DateTime('2015-12-31 17:59:59').asdatetime()
datetime.datetime(2015, 12, 31, 17, 59, 59)
>>> DateTime("2004/12/30 00:20:00 GMT+1").asdatetime()
datetime.datetime(2004, 12, 30, 0, 20, tzinfo=<StaticTzInfo 'GMT+1'>)
```

alternatively we'd have to make the format configurable :confused: